src/Screens/Flows/App/AppsScreen/Components/FavoritesBottomSheet.tsx

```diff
@@ -167,9 +171,13 @@ export const FavoritesBottomSheet = React.forwardRef<BottomSheetModalMethods, Pr
                         windowSize={5}
                         ListEmptyComponent={
                             <ListEmptyResults
-                                subtitle={LL.FAVOURITES_DAPPS_NO_RECORDS()}
-                                icon={"icon-search"}
+                                subtitle={LL.FAVOURITES_DAPPS_EMPTY_LIST()}
+                                icon={"icon-alert-circle"}
+                                iconColor={theme.colors.mnemonicCardBorder}
```

Comment:
I think color is off
I see that on Figma it's Gray 400, but `mnemonicCardBorder` is 300 in light mode